unit1/lesson12.rst

```diff
@@ -0,0 +1,41 @@
+Unit 1 - Lesson 12 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Create static HTML pages using a variety of different tags inside the body.
+- Create an account on Github and create hello world repository.
+- Understand what Git is and why programmers use it.
+
+Objectives
+----------
+
+HTML
+^^^^
+
+- Know what HTML stands for
+- Describe the components of HTML tag structure
+- Describe what an HTML element is, and how an element is related to a tag
+- Name, describe the purpose of, and use common HTML tags: <b>, <em>, <a>, <img>, <br>, <p>, <span>, <div>, <form>
```

Comment:
I think in HTML 5, the <hr> tag became a thematic break between multiple topics. I do know that some people don't particularly like the underline because it can be confused very easily for a hyperlink.